src/homeork_2/Model/StudentUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.second.Model;
+
+public class StudentUtil {
+    private CustomArrayList arrayListOfStudents = new CustomArrayList();
+    private DoubleLinkedList doubleLinkedListOfStudents = new DoubleLinkedList();
+
+    public void initArrayList(Student... students) {
+        for (int i = 0; i < students.length; i++) {
+            arrayListOfStudents.add(students[i]);
+        }
+        printInitializedArrayList(arrayListOfStudents);
+    }
+
+    public void initLinkedList(Student... students) {
```

Comment:
if you created util class which has init methods for your collections why don't you move to create students and putting in your collections parts from the main method to here? 
I will not recommend to put them in CustomArrayList or CustomLinkedList classes, but in utils class it is  ok(and even recommended)